src/Symfony/Component/Translation/Tests/TranslatorTest.php

```diff
@@ -683,6 +683,20 @@ public function testIntlFormattedDomain()
         $this->assertSame('Hi Bob', $translator->trans('some_message', ['%name%' => 'Bob']));
     }
 
+    /**
+     * @requires extension intl
```

Comment:
This test does not require the intl extension. The test will also work when using `symfony/polyfill-intl-icu` (and the testsuite always has it available)